platform/platform-sportpaper/src/main/java/tc/oc/pgm/platform/sportpaper/material/SpColorUtils.java

```diff
@@ -71,6 +71,11 @@ public boolean isColor(BlockState block, DyeColor color) {
     return color.getWoolData() == block.getRawData();
   }
 
+  @Override
+  public boolean isColor(ItemStack stack, DyeColor color) {
+    return color.getWoolData() == stack.getDurability();
```

Comment:
If this is called on other things with data / durability this may return true even if color is not relevant.
Not sure if this is currently an issue based off of current usage of the method, but it seems a candidate for easily introducing bugs if kept as is.
If you double check current usage and see no issues, then I'm fine with keeping it as is for this pr.

Reply:
Currently, this method is used only here: (the mentioned method is called by other 4 callers, but the preceding wool query filters that away)

https://github.com/PGMDev/PGM/blob/0065f1527c2e85683f66d6d2d2d6f8c0b69b6aa1/core/src/main/java/tc/oc/pgm/wool/MonumentWoolFactory.java#L123-L125

As for this method being called for un-colorable items, I agree this might open a possibility for other bugs, but none of the other methods in legacy ColorUtils check for whether the block can be colored either – fixing this should go in another PR as it would be out of scope of this PR and #1363 this PR is set to be merged into.